bitwarden_license/bit-web/src/app/secrets-manager/settings/porting/sm-export.component.html

```diff
@@ -0,0 +1,20 @@
+<sm-header></sm-header>
+
+<form [formGroup]="formGroup" [bitSubmit]="submit">
+  <div class="tw-my-4 tw-max-w-xl">
+    <app-callout type="info" title="{{ 'exportingOrganizationSecretDataTitle' | i18n }}">
+      {{ "exportingOrganizationSecretDataDescription" | i18n: orgName }}
+    </app-callout>
+  </div>
+
+  <bit-form-field class="tw-max-w-sm">
+    <bit-label>{{ "fileFormat" | i18n }}</bit-label>
+    <select bitInput required formControlName="format">
```

Comment:
This is applied by the validators.

```suggestion
    <select bitInput formControlName="format">
```

Reply:
Removed